src-vue/src/views/SettingsView.vue

```diff
@@ -26,6 +28,11 @@ import { ElNotification } from 'element-plus';
 
 export default defineComponent({
     name: "SettingsView",
+    computed: {
+        flightcoreVersion(): string {
+            return this.$store.state.flightcore_version; // TODO change to FlightCore version
```

Comment:
Still TODO? :)

Reply:
Whoops, no. Removed in 0044e58e0df0bc341e2fbbe28ce3b34985ae51b0